src/lib/style/components/_cards.scss

```diff
@@ -28,6 +28,9 @@ $text-color-lighter: #898989;
   margin-top: 30px; // just for test page
   margin-bottom: 30px; // just for test page
 
+  .SRC-portalCardMain {
+    display: flex;
```

Comment:
Based on the images I don't think it would make a difference as the cards are fairly crowded, but consider specifying `justify-content` (unless the default `flex-start` is desired).